internal/test/testdata/engine/case_23.yaml

```diff
@@ -51,7 +51,7 @@ wantOutputs:
               "scope": "acme.hr",
             },
         },
-      "effectiveDerivedRoles": ["any_employee", "employee_that_owns_the_record"],
+      "effectiveDerivedRoles": ["employee_that_owns_the_record"],
```

Comment:
Why is `any_employee` not in the list any more? Looks like it should?

Reply:
Oh! I think GitHub's showing me the diff against your previous commits in this PR 🫤 
[This file on `main`](https://github.com/cerbos/cerbos/blob/41787ba1b9e47fdabc9ab15a5b03b3de4807b5b2/internal/test/testdata/engine/case_23.yaml#L54) doesn't have `any_employee` either.

Reply:
Lol. I too have been fooled because I remember investigating one at length, and assumed it was this one because the file has a diff, but the diff is a comma 🤦‍♂️.

Nice to know it shouldn't be there 😆